test/parallel/test-stream2-httpclient-response-end.js

```diff
@@ -11,7 +11,9 @@ const server = http.createServer(function(req, res) {
     let data = '';
     res.on('readable', common.mustCall(function() {
       console.log('readable event');
-      data += res.read();
+      let input = res.read();
+      if (input)
```

Comment:
This doesn't sound like a healthy situation...
The event is named `'readable'`, so `res` should be readable, and IMHO that means `res.read()` should not be falsy.